### PR TITLE
[8169] aplus translations for release

### PR DESCRIPTION
**Pull request by m4ra**
**Tasks**
- [x] PR name contains story or task reference
- [x] Documentation (docs and inline)
- [ ] Tests (including n+1 and django_assert_num_queries where applicable)
- [ ] Changelog

### CODE REVIEW

**Comment by m4ra** [comment]
@goapunk this is the stage of having pushed to transifex, but I haven't pulled yet. It is not clear from the documentation, if we need to merge this one, then create a new PR to pull from transifex?
https://liqd.github.io/adhocracy-plus/languages_and_translations/
I am at the end of `Push new source strings to transifex` section.

File: docs/languages_and_translations.md [hunk 1]
```diff
@@ -7,21 +7,32 @@ The translations for the main branch are done in transifex. If you want to trans
...
 
-`make mo`
+`make mo`  
 
 Push updated source file with new strings to transifex to be be translated (Note: commented out strings will not be ignored by transifex, but handled as normal source strings, so remove them before from source files)
 
-(If you do this the first time, you might need to [generate an API token](https://docs.transifex.com/account/authentication) and add it in the terminal. It will be automatically stored in your `~/.transifexrc`)
 
-`tx push -s`
+`tx push`
```
Comment (goapunk) [inline docs/languages_and_translations.md:34]: do we not need the `-s` here?

Comment (m4ra) [reply docs/languages_and_translations.md:34]: No it doesn't exist in their current documentation, and it worked without for me, with a fresh tx install

Comment (goapunk) [reply docs/languages_and_translations.md:34]: ah: > when neither -s/-t are set, -s is assumed

**Comment by goapunk** [comment]
> @goapunk this is the stage of having pushed to transifex, but I haven't pulled yet. It is not clear from the documentation, if we need to merge this one, then create a new PR to pull from transifex? https://liqd.github.io/adhocracy-plus/languages_and_translations/ I am at the end of `Push new source strings to transifex` section.

We now have to wait until the translations are done. I guess it doesn't really matter if we merge this now and make a new PR once they are done or keep this open until we pulled them from transifex

**Comment by m4ra** [comment]
> We now have to wait until the translations are done. I guess it doesn't really matter if we merge this now and make a new PR once they are done or keep this open until we pulled them from transifex

I see, we can wait and pull in the same branch. Do we check periodically if they have finished and how?

**Review by goapunk — APPROVED** [review]
LGTM

**Pull request merged**